.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
 
 php:
-  - 5.6
   - 7.0
   - 7.1
-  - nightly
+  - 7.2
```

Comment:
'nightly' should still be on the list to test against future versions.
You might also want to change the PHP versions checker configuration of grumphp.util.phpversion in resources/config/util.yml of 7.2 to 30 Nov 2020.